syntax/crystal.vim

```diff
@@ -313,8 +313,8 @@ if !exists('b:crystal_no_expensive') && !exists('g:crystal_no_expensive')
   SynFold 'select' syn region crystalSelectExpression      matchgroup=crystalConditional start="\<select\>"            end="\<end\>" contains=TOP
   SynFold 'if'     syn region crystalConditionalExpression matchgroup=crystalConditional start="\%(\%(^\|\.\.\.\=\|[{:,;([<>~\*/%&^|+=-]\|\%(\<[_[:lower:]][_[:alnum:]]*\)\@<![?!]\)\s*\)\@<=\%(if\|ifdef\|unless\)\>" end="\%(\%(\%(\.\@1<!\.\)\|::\)\s*\)\@<!\<end\>" contains=TOP
 
-  syn match crystalConditional "\<\%(then\|else\|when\)\>[?!]\@!" contained containedin=crystalCaseExpression
-  syn match crystalConditional "\<\%(when\|else\)\>[?!]\@!" contained containedin=crystalSelectExpression
+  syn match crystalConditional "\<\%(then\|else\|when\|in\)\>[?!]\@!" contained containedin=crystalCaseExpression
+  syn match crystalConditional "\<\%(when\|in\|else\)\>[?!]\@!" contained containedin=crystalSelectExpression
```

Comment:
This line is for `select` statement. Is `select ... in ... end` a valid syntax in Crystal?

Reply:
woops! nice catch